app/views/search/results.njk

```diff
@@ -49,10 +49,18 @@
         page: currentPage,
         searchString: searchString
       }) }}
-      {{ govukPagination({
-        previous: previous,
-        next: next
-      }) }}
+
+     {{ govukPagination({
```

Comment:
According to the suggestion in Accessibility Audit, the labelText and the href passed on from the previous and next variables are fine. The suggestion is basically asking us to add an aria-label to the whole of pagination component, the next option and then previous option.
For the pagination component, within the results.njk modify the govukpagination and add the following object
attributes: {
 aria-label: 'Pagination Navigation'
}
In order to add the aria-label to previous and next options:
These 2 objects are defined in a function called getPaginationAttributes within routes/search/results.ts, and are passed into govukpagination.
So in order to set the attributes for next and previous, go to getPaginationAttributes function and if you find the variables called previous and next, they already have an attribute object defined so you can just add the aria-label field to it and it will be reflected onto the fe component.

Reply:
changes are done as per suggestion.